section3/arrayMethods.md

```diff
@@ -0,0 +1,15 @@
+//
+pop() - Removes the element at the end of the array.
+
+.push - Adds an element to the end of the array.
+
+.shift - Removes the element  at the beginning of the array.
+
+.unshift - Adds an element to the beginning of the array.
+
+.length - Shows the number of elements in the array.
```

Comment:
technically, length is a property (attribute). We can think of objects (an array is an object) as having state and behavior. The `length` relates to `Array state`, whereas all of the other examples you give here are methods—`Array behavior` in other words.

Just food for thought—lots more coming on this in Mod 1 as you learn more about object oriented programming.